test/object-store/object.cpp

```diff
@@ -2412,4 +2388,46 @@ TEST_CASE("Asymmetric Object") {
     }
 }
 
+TEST_CASE("KeyPath generation - star notation") {
+    Schema schema{
```

Comment:
It would probably be worth it to have a single backlink property somewhere since the resolver logic also supports this.